src/clr-angular/data/datagrid/datagrid.ts

```diff
@@ -168,16 +199,25 @@ export class ClrDatagrid implements AfterContentInit, AfterViewInit, OnDestroy {
      */
 
     @ContentChildren(ClrDatagridRow) rows: QueryList<ClrDatagridRow>;
+    @ViewChild("scrollableColumns", {read: ViewContainerRef}) scrollableColumns: ViewContainerRef;
 
     ngAfterContentInit() {
+        if (!this.items.smart) {
+            this.items.all = this.rows.map((row: ClrDatagridRow) => row.item);
+        }
+
+        this.rows.forEach(row => {
+            this.displayedRows.insert(row.view);
+        });
+
         this._subscriptions.push(this.rows.changes.subscribe(() => {
```

Comment:
Another place where perhaps we want the row changes to be a behavior subject so we trigger this on init and don't have duplicated logic in rows above.

Reply:
Updated when I added behavior subject for similar code in datagrid-row.